### ISSUE TITLE
"Unable to complete exercise" when clicking "Mark as complete" on a passed project. 

### ISSUE THREAD

**Issue by tvalesky** [exercism/exercism#6160]
I have completed a few projects today. For the last 2 projects, after the project passed all test cases, I clicked the "Mark as complete" button and received a message in the dialog saying "Unable to complete exercise".  The projects both remain in 'In-progress' status.

**Comment by SleeplessByte** [comment]
Is your Exercism username also @tvalesky ?

**Comment by tvalesky** [comment]
Yes

**Comment by tvalesky** [comment]
Out of curiosity, I switched from the Python track to the Java track, completed exercise 1, got it to pass the tests, and tried "Mark as Complete." I still get "Unable to complete the exercise."

**Comment by iHiD** [comment]
So weirdly there's no logged errors for this for you (or for completing solutions in general). Any chance you could record a screen capture (Loom is good for this) so I can see exactly what's happening and try and spot some clues? 

Also, can you please check you don't have any browser extensions that could be interfering with this somehow?

Thanks.

**Comment by tvalesky** [comment]
Can't do a screen capture. But there's nothing notable on the screen,
except the error message.

Before I started having the issue, I successfully completed 29 exercises in
the Python track. I didn't make any config changes between the successful
submissions and the failed ones.

I tried it from a machine outside the VPN i was on, and it worked. I tried
it again on a machine within the VPN, and it failed. So it looks like it's
some sort of issue that's specific to the VPN.

Near-term workaround is that I can just use the outside-the-VPN machine to
mark things as complete. I'm not sure that there's anything to be done
beyond that.
